src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -62,6 +62,13 @@ public ResponseEntity<Map<String, Boolean>> getDraftStatuses(
     return ResponseEntity.status(HttpStatus.CREATED).body(results);
   }
 
+  @GetMapping("/measures/byMeasureSetId")
+  public ResponseEntity<List<MeasureListDTO>> getMeasuresByMeasureSetId(
+      @RequestParam(name = "measureSetId") String measureSetId) {
+    List<MeasureListDTO> results = measureService.getMeasuresByMeasureSet(measureSetId);
+    return ResponseEntity.status(HttpStatus.CREATED).body(results);
```

Comment:
It looks like MeasureService is used and MeasureSearchService is not used anywhere?

Reply:
also `HttpStatus.CREATED` is not appropriate here

Reply:
Updated structure. Added correct statusCode for a get.